### ISSUE TITLE
Training Stuck at 0%

### ISSUE THREAD

**Issue by momo1986** [keras-team/autokeras#715]
Autokeras Version: legacy-0.4.0
Python version: 3.6.7
Tensorflow version: 1.13.0
Keras Version:2.2.4

On my own datasets with label regression tasks, here is the problem
![image](https://user-images.githubusercontent.com/34225874/61504414-d668cd00-aa0d-11e9-9d5e-26e5d82ecb69.png)

Is there the potential to fix it?
Regards

**Comment by haofanwang** [comment]
I met with the same problem when I played with the example.

**Comment by haofanwang** [comment]
I met the same problem before. I just close the terminal and rerun the code, and the training process can work well surprisedly.

**Comment by momo1986** [comment]
Hello, @haofanwang, I cannot work with your suggestion. However, still be grateful to you.

By the way, what is your label format, is it a classification problem or a regression problem?

Thanks & Regards!

**Comment by momo1986** [comment]
Same Regression Problem：
Using CnnModule instead of ImageRegressor,  still stuck at 0% problem.
Thanks & Regards!

**Comment by momo1986** [comment]
Attached problem with image:
![image](https://user-images.githubusercontent.com/34225874/61679124-c287d880-ad37-11e9-9680-0fe0a88eab28.png)

**Comment by themohitaggarwal** [comment]
Same thing happened with me. Shutdown the kernel and try rerunning the code. Worked for me.

**Comment by ji3g4m6zo6** [comment]
Same Problem.

Autokeras Version: 0.4.0
Python version: 3.6.7
Tensorflow version: 1.13.1
Keras Version:2.3.0
CUDA Version: 10.1
Driver Version: 430.40
GPU: RTX 2080 Super


![image](http://cucugi.us/images/autokeras_error.gif)
